### PR TITLE
add practice better

### LINKED ISSUE #6711: Add Practice Better
### Site name

Practice Better

### Site URL

https://practicebetter.io/

### Documentation URL

https://help.practicebetter.io/hc/en-us/articles/360055519791-Using-Two-Factor-Authentication

### Supported 2FA methods

- [X] TOTP (RFC-6238, Google Authenticator)
- [ ] Proprietary Software (Authy, Duo, etc.)
- [ ] FIDO2/U2F/WebAuthn security keys
- [ ] Proprietary hardware (Yubico OTP, RSA keys, etc.)
- [ ] Phone calls
- [ ] SMS tokens
- [ ] Email tokens

### Eligibility

- [X] I have checked that the site meets 2fa.directory's [contributing guidelines and site criteria](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md)
- [X] The site is ranked within the top 200,000 sites on [Similarweb](https://www.similarweb.com/) or has a substantial social media following.
- [X] The site does not contain or promote content that violates 2fa.directory's [excluded categories and websites guidelines](https://github.com/2factorauth/twofactorauth/blob/master/EXCLUSION.md), including pornographic, discriminatory, or unlawful content.
- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by person1234-ux**
resolves #6711

### CODE REVIEW

**Comment by person1234-ux** [comment]
resolves #6711

File: entries/p/practicebetter.io.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "practice better": {
+    "domain": "practicebetter.io",
+
```
Comment (phallobst) [inline entries/p/practicebetter.io.json]:
```suggestion
```

File: entries/p/practicebetter.io.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "practice better": {
+    "domain": "practicebetter.io",
+    
+    "img": "practicemore.io.png",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.practicebetter.io/hc/en-us/articles/360055519791-Using-Two-Factor-Authentication-with-Practice-Better",
```
Comment (phallobst) [inline entries/p/practicebetter.io.json]:
```suggestion
    "documentation": "https://help.practicebetter.io/hc/en-us/articles/360055519791",
```

File: entries/p/practicebetter.io.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "keywords": [
+      "health",
+      "food"
```
Comment (phallobst) [inline entries/p/practicebetter.io.json]:
```suggestion
      "health"
```

**Review by phallobst — CHANGES_REQUESTED** [review]
For the logo, you could use one of these
practicebetter io long

practicebetter io short
:

File: entries/p/practicebetter.io.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "practice better": {
```
Comment (phallobst) [inline entries/p/practicebetter.io.json]:
```suggestion
  "Practice Better": {
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**